en/mkdocs.yml

```diff
@@ -2160,6 +2166,5 @@ extra:
         - type: linkedin
           link: https://www.linkedin.com/company/wso2
     site_version: 4.0.0
-    #base_path: http://localhost:8000
```

Comment:
Shall we revert this? It's commented out for ease of use when checking locally.

Reply:
Fixed